setup/stack/states/ilias-update-hook/ilias-update-hook/update_hook.php.j2

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2024 - Daniel Weise <daniel.weise@concepts-and-training.de> - Extended GPL, see LICENSE */
+
+$request_headers = apache_request_headers();
+if (is_null($request_headers) || !array_key_exists("Authorization", $request_headers)) {
+    http_response_code(401);
+    throw new Exception("Missing Authorization header");
+}
+$request_token = $request_headers["Authorization"];
+
+$environment_token = getenv("UPDATE_TOKEN");
+if (is_null($environment_token)) {
+    http_response_code(401);
+    throw new Exception("Missing Environment Token");
+}
+
+if ($environment_token !== $request_token) {
+    http_response_code(401);
+    throw new Exception("Invalid Token");
```

Comment:
Bitte hier jeweils nur "Authorization required" zurückmelden in allen Fällen.

Reply:
Done!